ts/test/session/unit/receiver/drop_incoming/shouldProcessMessage_test.ts

```diff
@@ -0,0 +1,130 @@
+import { describe } from 'mocha';
+import Sinon from 'sinon';
+import Long from 'long';
+import { expect } from 'chai';
+import { TestUtils } from '../../../../test-utils';
+import { shouldProcessContentMessage } from '../../../../../receiver/common';
+import { DURATION } from '../../../../../session/constants';
+
+describe('shouldProcessContentMessage', () => {
+  let envelopeTs: number;
+  beforeEach(() => {
+    TestUtils.stubWindowLog();
+    envelopeTs = Math.floor(Date.now() + Math.random() * 1000000);
+  });
+
+  afterEach(() => {
+    Sinon.restore();
+  });
+
+  describe('not a community', () => {
+    const isCommunity = false;
+    describe('with sig timestamp', () => {
+      it('if timestamps match: return true', async () => {
+        expect(
+          shouldProcessContentMessage(
+            { timestamp: envelopeTs },
+            { sigTimestamp: envelopeTs },
+            isCommunity
+          )
+        ).to.eq(true);
+      });
+      it('if timestamps do not match: return false', async () => {
+        expect(
+          shouldProcessContentMessage(
+            { timestamp: envelopeTs },
+            { sigTimestamp: envelopeTs + 2 },
+            isCommunity
+          )
+        ).to.eq(false);
+      });
+    });
+    describe('without sig timestamp', () => {
+      it('if timestamps match or not: return true', async () => {
```

Comment:
Should this not be false? Or otherwise update the it description to explain it's for backwards compat